app/auth/jwtTokensLib.js

```diff
@@ -0,0 +1,37 @@
+const jwt = require('jsonwebtoken'); 
+const { sendError } = require('../util/error');
+
+// Following library is used for generating token
+// based on user detail
+exports.createToken = (user) => {
+    const accessToken = jwt.sign( 
+    { user: user._id }, 
```

Comment:
Planning to encrypt UserId in upcoming PR.

Reply:
Sounds good